cloudferry_devlab/cloudferry_devlab/generate_load.py

```diff
@@ -265,15 +266,21 @@ def update_filtering_file(self):
             vm_id_list = [x.__dict__['id'] for x in
                           self.novaclient.servers.list()]
 
-            img_id_list = [x.__dict__['id'] for x in
-                           self.glanceclient.images.list(search_opts={
+            img_name_list = [x.__dict__['name'] for x in
+                             self.glanceclient.images.list(search_opts={
                                'is_public': False})]
 
             for key in filter_dict.keys():
                 if key == 'images':
-                    for img_id in img_id_list:
-                        if img_id not in not_incl_img:
-                            filter_dict[key]['images_list'].append(str(img_id))
+                    for img_name in img_name_list:
+                        if (img_list == 'exclude_images_list' and img_name
+                                in self.config.images_to_be_blacklisted) or \
+                                (img_list == 'images_list' and img_name not in
+                                 self.config.images_not_included_in_filter and
+                                 img_name not in
+                                 self.config.images_to_be_blacklisted):
+                            filter_dict[key][img_list].append(
```

Comment:
Actually I don't see any difference between images_to_be_blacklisted and images_not_included_in_filter... shouldn't we use kind of black_list section in filter file? If don't then just keep the same logic as for 'not included in filter'